spotify_player/src/ui/page.rs

```diff
@@ -567,12 +567,19 @@ pub fn render_commands_help_page(frame: &mut Frame, ui: &mut UIStateGuard, rect:
     let help_table = Table::new(
         map.into_iter()
             .skip(scroll_offset)
-            .map(|(command, keys)| {
+            .enumerate()
+            .map(|(i, (command, keys))| {
                 Row::new(vec![
                     Cell::from(format!("{command:?}")),
                     Cell::from(format!("[{keys}]")),
                     Cell::from(command.desc()),
                 ])
+                    // adding alternating row colors
+                    .style(if (i + scroll_offset) % 2 == 0 {
+                        ui.theme.app().bg(Color::DarkGray)
```

Comment:
should be a component style: https://github.com/aome510/spotify-player/blob/master/docs/config.md#component-styles

Reply:
Did you have a particular style in mind? Almost seems to me like having a new alternate row field for the ComponentStyle struct could work.

Reply:
yeah, I think you could add new component style named `help_table_alternating_row` for example. It's to allow people with incompatible theme to update the style.